Create scripts to use jabba with nushell when installing jabba

pohart opened the pull request:
Fixes #59

patrick-mccourt commented:
Thanks for submitting a PR for this, hadn't heard of nushell before but looks interesting. 

I've tried your change out in both Linux and Windows. Linux seems to be working fine but on Windows I'm getting some errors:

I'm using Windows Terminal and nushell to try this out. the jabba.nu file seems to get created and when I try to install a jdk it downloads, but it errors when trying to run a cat command. I might be doing it wrong but I thought I'd check before merging

![Screenshot 2025-03-28 213625](https://github.com/user-attachments/assets/d3475149-e6c3-4980-aa65-3be3be05649e)
![Screenshot 2025-03-28 213555](https://github.com/user-attachments/assets/a8cd49f9-7269-403e-aaeb-ea15f559cb8f)

pohart commented:
No you're right. I had cat installed in my system and didn't notice it wasn't the right command. I've got another computer I can use that doesn't have the unix utilities installed. I'll get it fixed up there.

pohart commented:
I modified both the bash script and ps1 to use "open" instead of "cat" for nushell.

I also noticed that if I didn't pass JABBA_HOME to install.sh it put a literal $HOME into the generated script. This doesn't work in nushell, so I replaced $HOME with ~ in nushell only.

patrick-mccourt commented:
LGTM
